### DISCUSSION TITLE
Creating a Question Dialog That Defaults to Cancel

### DISCUSSION THREAD

**Discussion by tataDan** [wailsapp/wails#1870] [Q&A]
The following code will open a question dialog:

```
func (a *App) Confirm() {
	selection, err := runtime.MessageDialog(a.ctx, runtime.MessageDialogOptions{
		Type:          runtime.QuestionDialog,
		Title:         "CONFIRM DELETION",
		Message:       "Are you sure you want to delete these records?",
	})
	if err != nil {
		fmt.Println(err)
	}
	fmt.Println(selection)
}
```
In Linux the "No" button will be the default button, but in Windows the "Yes" button will be the default.  Is it possible to make the "No" button the default on Windows?

Is should be noted the [wails documention](https://wails.io/docs/reference/runtime/dialog) contains this text:
```
Windows

Windows has standard dialog types in which the buttons are not customisable. The value returned will be one of: "Ok", "Cancel", "Abort", "Retry", "Ignore", "Yes", "No", "Try Again" or "Continue"

Linux

Linux has standard dialog types in which the buttons are not customisable. The value returned will be one of: "Ok", "Cancel", "Yes", "No"
```
However, there are cross-platform libraries for displaying question dialogs that allow the developer to make the "No" button the default on Windows.

**Comment by leaanthony** [comment]
Not currently supported. Please open a feature request with a proposal for the change and link documentation that can make it happen. If you really want it sooner rather than later, please feel free to open a PR for it 👍
